packages/mongo/tests/hooks/meteor_1_4_id_object.js

```
@@ -0,0 +1,54 @@
+import { Meteor } from 'meteor/meteor'
+import { Mongo } from 'meteor/mongo'
+import { Tinytest } from 'meteor/tinytest'
+
+const collection = Meteor.users
+const collection1 = new Mongo.Collection('test_insert_mongoid_collection1', { idGeneration: 'MONGO' })
+
+if (Meteor.isServer) {
+  collection.allow({
+    insertAsync: function () { return true },
```

ESLint

[error] Expected method shorthand.

(object-shorthand)